code/controllers/subsystem.dm

```diff
@@ -163,11 +169,20 @@
 /datum/controller/subsystem/Initialize(start_timeofday)
 	initialized = TRUE
 	SEND_SIGNAL(src, COMSIG_SUBSYSTEM_POST_INITIALIZE, start_timeofday)
-	var/time = (REALTIMEOFDAY - start_timeofday) / 10
-	var/msg = "Initialized [name] subsystem within [time] second[time == 1 ? "" : "s"]!"
-	to_chat(world, span_boldannounce("[msg]"))
+	var/time = (REALTIMEOFDAY - start_timeofday)
```

Comment:
just divide by 10 here, i don't see you using the straight up time value anywhere